### PR TITLE
Normalization from example_ynh

### PR DESCRIPTION
**Pull request by maniackcrudelis**
## Problem
- *Global upgrade of the package.*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Kay0u
- [x] **Approval (LGTM)** : Kay0u
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/wordpress_ynh%20PR64/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

File: scripts/install [hunk 1]
```diff
@@ -116,49 +112,49 @@ ynh_add_fpm_config
...
-ynh_replace_string "__DB_USER__" "$db_name" $final_path/wp-config.php
-ynh_replace_string "__DB_PWD__" "$db_pwd" $final_path/wp-config.php
+ynh_replace_string --match_string="__DB_USER__" --replace_string=$db_name --target_file=$final_path/wp-config.php
+ynh_replace_string --match_string="__DB_PWD__" --replace_string="$db_pwd" --target_file=$final_path/wp-config.php
 
 for i in 1 2 3 4 5 6 7 8
 do
-	j=$(dd if=/dev/urandom bs=1 count=200 2> /dev/null | tr -c -d '[A-Za-z0-9]' | sed -n 's/\(.\{40\}\).*/\1/p')
+	j=$(ynh_string_random --length=40)
 	if [ "$j" = "" ];
```
Comment (kay0u) [inline scripts/install]: Maybe this **if** is no longer necessary

Comment (maniackcrudelis) [reply scripts/install]: That's a very old part of the code, should be tried.

**Review by kay0u — DISMISSED** [review]
LGTM & code review

File: scripts/install (hunk 1 above)
Comment (kay0u) [reply scripts/install]: indeed

Comment (kay0u) [reply scripts/install]: [Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/wordpress_ynh%20(Kay0u)/)

Comment (kay0u) [reply scripts/install]:
```
Info:  [######..............] > Configuring wordpress...
Info:  for loop: SpJzljnlJhEMbRV5Qg4kDUmjNBbh6B7DiXamLPjs
Info:  for loop: dPqaPZoIZY4hpjBHxdGMYWO7Z9MKIZB6QOTYBZh7
Info:  for loop: Dq09CUXixCdRc1cqjEsEgnywhmjjPk1lj77BIZu1
Info:  for loop: SPGavq4ARYbY11zrrrwoKbLDZ2XplJKkotCW8jgT
Info:  for loop: qmrdAT4UV8lButEPO4fs9m86su0CSfsO4wLeCYVW
Info:  for loop: GPmqdxoZf2GSRshGtGao2aCTWV8OE7B9DruW0aox
Info:  for loop: riWusw0uF89firJTU8sL6Z9XL41aCrWkWVZx5Hxz
Info:  for loop: ZegzgwMsfr4E5Z5f1A6pv8CUvzyrSuJpPMq8pIVj
Info:  [######..............] > Installing wordpress with Curl...
```

For now it's ok... But maybe it's hard to reproduce this bug :-/

Code added 6 years ago:

https://github.com/YunoHost-Apps/wordpress_ynh/commit/802da8386b63e29d841935805cd952b64e6af222

Comment (maniackcrudelis) [reply scripts/install]: I guess a simple `sleep 0.5` between each loop would be enough to be sure.

**Review by JimboJoe — DISMISSED** [review]
LGTM :+1:

**Comment by JimboJoe** [comment]
Can be merged in 3 days.

**Comment by kay0u** [comment]
I don't think this sleep is necessary, but if you're more confident with it.

We can still merge in 3 days...

**Pull request merged**